docs/build/micro-rdk/board/esp32.md

```diff
@@ -86,7 +86,7 @@ Copy the following JSON template and paste it into your configuration inside the
 {{< /tabs >}}
 
 Edit and fill in the attributes as applicable.
-Click **Save config**.
+Press **Command+S** to save your config.
```

Comment:
Is there no save button anymore? Not everyone is on a mac so if we need to do this we need to also offer the windows/linux equivalent...

Reply:
Oh that is a very good point. There is a save button. I'll change the command to that